ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -151,46 +151,107 @@ def get_partial_event_contingency_tables(self) -> List[EventContingencyTable]:
         query = f"""
             WITH 
                 funnel_people AS ({funnel_persons_query}),
+                non_deleted_distinct_ids AS (
+                    SELECT distinct_id,
+                        argMax(person_id, _timestamp) as person_id
+                    FROM (
+                        SELECT distinct_id,
+                                person_id,
+                                max(_timestamp) as _timestamp
+                            FROM person_distinct_id
+                            WHERE team_id = {self._team.pk}
```

Comment:
How might I know that this is available btw?

tbh I think it's more understandable like this rather than relying on the params that are coming from another class. The only think I think we should rely on `funnel_persons_params` for is anything in `funnel_persons_query`.